app/fonts/spinnaker/icons.css

```diff
@@ -62,8 +71,8 @@
   content: '\e902';
 }
 .icon-sitemap:before {
-  content: '\e900';
```

Comment:
Hmm I dont think these should have changed. I wonder why icomoon reassigned them?

Reply:
When I moved the newly imported icons down to the collection of existing icons, icomoon put them in the front, so maybe it's because of that?